.github/workflows/release.yml

```
@@ -0,0 +1,171 @@
+        echo '```bash' >> release_notes.md
+        echo "git clone https://github.com/biwakonbu/ai-pr-review-checker.git" >> release_notes.md
+        echo "cd ai-pr-review-checker" >> release_notes.md
+        echo "git checkout ${{ steps.version.outputs.version }}" >> release_notes.md
+        echo "go build -o gh-review-task ." >> release_notes.md
+        echo '```' >> release_notes.md
+        echo "" >> release_notes.md
+        echo "## Verification" >> release_notes.md
+        echo "Binary checksums are provided in the \`SHA256SUMS\` file." >> release_notes.md
+
+    - name: Create GitHub Release
+      uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)